code/Common/Compression.h

```diff
@@ -56,7 +63,7 @@ class Compression {
 
     /// @brief  Will open the access to the compression.
     /// @return true if close was successful, false if not.
-    bool open();
+    bool open(Format format);
```

Comment:
```suggestion
    bool open(Format format = Format::Binary);
```
A default of Format::Binary might be convenient for the most common use case.

Reply:
I prefer to support such options as an explicit parameter. It makes clear what the intention is behind that.